package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-account-link-extension",
-  "version": "2.1.1",
+  "version": "2.1.3",
```

Comment:
rebase this with `rc` and we'll release this whole thing as `2.2.0` (that PR has merged)